### PR TITLE
skeleton for helm-dashboard-v2

### PR DESCRIPTION
**Pull request by aman1376**
## Changes Proposed

Created the skeleton for the dashboard with mock data

Screenshot (51)

## Check List

- [ x] The title of my pull request is a short description of the changes
- [x ] This PR relates to some issue: 
- [ x] I have documented the changes made (if applicable)
- [ ] I have covered the changes with unit tests

### CODE REVIEW

**Comment by aman1376** [comment]
@chad11111 please have a look at this.

File: dashboard/src/App.tsx [hunk 1]
```diff
@@ -15,31 +15,115 @@ function App(): JSX.Element {
...
+    //         If you have trouble, don't hesitate to ask for help.
+    //       </p>
+    //     </div>
+    //     <div className="flex items-center justify-end px-4 py-2 bg-gray-100">
+    //       <a
+    //         className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded"
+    //         href="http://localhost:6006"
+    //       >
+    //         Run Storybook
+    //       </a>
```
Comment (chad11111) [inline dashboard/src/App.tsx:42]: remove the commented code plz.

File: go.sum [hunk 2]
```diff
@@ -18,6 +20,13 @@ cloud.google.com/go v0.74.0/go.mod h1:VV1xSbzvo+9QJOxLDaJfTjx5e+MePCpCWwvftOeQmW
 cloud.google.com/go v0.78.0/go.mod h1:QjdrLG0uq+YwhjoVOLsS1t7TW8fs36kLs4XO5R5ECHg=
 cloud.google.com/go v0.79.0/go.mod h1:3bzgcEeQlzbuEAYu4mrWhKqWjmpprinYgKJLgKHnbb8=
 cloud.google.com/go v0.81.0/go.mod h1:mk/AM35KwGk/Nm2YSeZbxXdrNK3KZOYHmLkOqC2V6E0=
+cloud.google.com/go v0.83.0/go.mod h1:Z7MJUsANfY0pYPdw0lbnivPx4/vhy/e2FEkSkF7vAVY=
+cloud.google.com/go v0.84.0/go.mod h1:RazrYuxIK6Kb7YrzzhPoLmCVzl7Sup4NrbKPg8KHSUM=
```
Comment (chad11111) [inline go.sum:24]: plz remove this from the pr.

File: dashboard/src/App.tsx [hunk 3]
```diff
@@ -15,31 +15,115 @@ function App(): JSX.Element {
...
+    //     </div>
+    //   </div>
+    // </div>
+    <div className="app">
```
Comment (chad11111) [inline dashboard/src/App.tsx:46]: please break these into separate smaller components like header, body, etc...

**Review by chad11111 — CHANGES_REQUESTED** [review]

File: go.sum (hunk 2 above)
Comment (chad11111) [reply go.sum:24]: im guessing the test is failing because of this.

**Comment by aman1376** [comment]
@chad11111 made the changes please have a look

File: dashboard/src/layout/Body.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import "../App.css"
+import Body_content from './Body_content'
+import Body_header from './Body_header'
+
+function Body():JSX.Element {
+  return (
+    <div className="card-right">
+          <Body_header/>
```
Comment (chad11111) [inline dashboard/src/layout/Body.tsx:8]: Please use camel case `BodyHeader`

**Pull request merged**